README.md

```diff
@@ -20,3 +20,43 @@ The plot with Shuxin's list generated from Lidia's and Taylor's papers:
 The plot with CCW's list but screening for all 25 codes:
 
 [ADRDtrend.pdf](https://github.com/ShuxinD/ADRDdata/files/8043033/ADRDtrend.pdf)
+
+## Data Dictionary 
+
+| Fieldname | Source |  Description | Role |
+| --------- | ------ | ------------ | ---- |
+| QID | Medicare | Person's ID | Cohort |
+| ADATE | Medicare | Admission date | Time |
+| DDATE | Medicare | Discharge date | Time |
+| zipcode_R | Medicare | Zipcode | Location |
+| DIAG 1-10 | Medicare | Diagnosis code | Outcome |
+| AGE | Medicare | Age | Confounder |
+| Sex_gp | Medicare | Sex | Confounder |
+| Race_gp | Medicare | Race | Confounder |
+| SSA_STATE_CD | Medicare | State code | Location |
+| SSA_CNTY_CD | Medicare | County code | Location |
+| PROV_NUM | Medicare | Hospital ID | Location |
+| ADM_SOURCE | Medicare | Admission source | Confounder |
+| ADM_TYPE | Medicare | Admission type (1 - Emergency, 2 - Urgent, 3 - Elective)) | Confounder |
+| Dual | Medicare | Eligible for both Medicare and Medicaid (1 - yes, 0 - otherwise) | Confounder |
+| Year | R | Year of hospital admission (from 2000 to 2016) | Time |
+| AD_primary | R | Do AD ICD codes apear as first diagnosis? (T/F) | Outcome |
+| AD_any | R | Is AD ICD codes in any of the DIAG 1-10? (T/F) | Outcome |
+| ADRD_primary | R | Do ADRD ICD codes appear as first diagnosis? (T/F) | Outcome |
+| ADRD_any | R | Is ADRD ICD code among DIAG 1-10? (T/F) | Outcome |
+
+|  | List of ICD codes (used as inclusion criteria) |
```

Comment:
As for the list of ICD codes to identify AD/ADRD: you can refer to line 11 where the source of the list is. You can modify the position. Basically, the list below is the same thing as line 11.